tests/functional/u/use/use_implicit_booleaness_not_len.py

```diff
@@ -35,7 +35,7 @@
 if 0 == len('TEST') < 10:  # Should be fine
     pass
 
-if 0 < 1 <= len('TEST') < 10:  # Should be fine
+if 1 <= len('TEST') < 10:  # Should be fine
```

Comment:
I feel like we're losing a bit of this test case. does it work if we just change '0' to 'z'?

Reply:
```suggestion
if  0 < 1 <= len('TEST') < 10:  # Should be fine
```
I agree with Jacob, we should not modify tests that are unrelated to the new check. This is potentially code that come from someone that created an issue after encountering a problem and we then merged this as a regression alongside a fix so there's a lot of work inside this line.